src/main/java/io/confluent/connect/jdbc/sink/dialect/SqliteDialect.java

```diff
@@ -36,6 +40,15 @@ public SqliteDialect() {
 
   @Override
   protected String getSqlType(String schemaName, Map<String, String> parameters, Schema.Type type) {
+    if (schemaName != null) {
+      switch (schemaName) {
+        case Decimal.LOGICAL_NAME:
+        case Date.LOGICAL_NAME:
+        case Time.LOGICAL_NAME:
+        case Timestamp.LOGICAL_NAME:
+          return "NUMERIC";
```

Comment:
@ewencp fixed in 104225afe78405b1f554f4462fcca4927acc50fd. Also included another change in that commit to map to `INTEGER` for integral types rather than `NUMERIC` (there seems to be some casting nuance in which they are different and `INTEGER` is more accurate anyway). Since we are anyway doing integration-style testing for SQLite also  folded in logical type conversions into an existing test.